packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -100,6 +101,12 @@ class CupertinoSwitch extends StatefulWidget {
   /// the [CupertinoTheme] in accordance to native iOS behavior.
   final Color activeColor;
 
+  /// The color to use for the background when the switch is off
+  ///
+  /// Defaults to [CupertinoColors.secondarySystemFill] when null and ignores
+  /// the [CupertinoTheme] in accordance to native iOS behavior.
```

Comment:
`CupertinoTheme` doesn't seem to have control over `CupertinoSwitch`'s appearance? I wonder why it is brought up here?

Reply:
It was just to indicate that this is not governed by the `CupertinoTheme`. Though, I removed it.